qa/tasks/cephadm.py

```diff
@@ -100,6 +100,43 @@ def update_archive_setting(ctx, key, value):
         yaml.safe_dump(info_yaml, info_file, default_flow_style=False)
 
 
+@contextlib.contextmanager
+def nvmeof_gateway_cfg(ctx, config):
+    source_host = config.get('source')
+    target_host = config.get('target')
+    nvmeof_service = config.get('service')
+    assert source_host
+    assert target_host
+    assert nvmeof_service
+    (remote,) = ctx.cluster.only(source_host).remotes.keys()
```

Comment:
```suggestion
    remote = ctx.cluster.only(source_host).remotes.keys()[0]
```

You could still hit an `IndexError` if `keys()` returns `[]`, but at least you won't get a `ValueError` if it returns a list of length >1

Reply:
Oh that's right! Thank you :)